package.json

```diff
@@ -26,8 +26,11 @@
     "@mdb/flora": "1.3.5",
     "@mdx-js/react": "2.2.1",
     "@sentry/nextjs": "7.36.0",
+    "@types/formidable": "2.0.5",
     "airtable": "0.11.6",
     "axios": "1.3.2",
+    "formidable": "2.1.1",
```

Comment:
no biggie, but I just wanted to ask why we are not locking into specific versions for our dependencies.

Reply:
this is locked into a specific version?